files/hipchat.rb

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env ruby
+
+require "#{File.dirname(__FILE__)}/base"
+#require_relative 'base'
```

Comment:
Heh. I still think that only supporting the sensu-embedded ruby would help for things like this :)